lib/occi/api/client/http/authn_plugins/keystone.rb

```diff
@@ -4,7 +4,7 @@ module AuthnPlugins
 
       class Keystone < Base
 
-        KEYSTONE_URI_REGEXP = /^(Keystone|snf-auth) uri='(.+)'$/
+        KEYSTONE_URI_REGEXP = /^(Keystone|snf-auth) uri="(.+)"$/
```

Comment:
@enolfc  So, will you update this regexp or should I? After merging the PR, I mean.